www/i18n/en.json

```diff
@@ -487,9 +343,7 @@
         "no-nrelop-start": "OPcode {{token}} does not start with 'nrelop'",
         "not-enough-parts": "OPcode {{token}} does not have at least three '_' characters",
         "invalid-subgroup": "Invalid OPcode {{token}}, subgroup {{subgroup}} not found in list {{config_subgroups}}",
-        "invalid-subgroup-no-default": "Invalid OPcode {{token}}, no subgroups, expected 'default' subgroup",
         "unable-download-config": "Unable to download study config",
-        "invalid-opcode-format": "Invalid OPcode format",
-        "error-loading-config-app-start": "Error loading config on app start"
```

Comment:
We might use this too. At least the base functionality is still supported, so I would double-check this.

Reply:
ah, this was a typo issue! I found a reference to `loading-config-app-start` and am updating it to refer to the `error-loading-config-app-start` key as well as adding it back (will add back in all languages)